### PR TITLE
Split documentation between essential and extra.

### PR DESCRIPTION
**Pull request by toots**

### CODE REVIEW

File: libs/hls.liq [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # @param ~id Force the value of the source ID.
 # @param ~reload How often (in seconds) the playlist should be reloaded.
 # @param uri Playlist URI
-# @flag experimental
+# @flags experimental
```
Comment (smimram) [inline libs/hls.liq]: did you really mean `@flags` here?

File: libs/playlist.liq [hunk 2]
```diff
@@ -4,6 +4,7 @@
 # reload function can optionally be called with a new playlist URI, otherwise,
 # it reloads the previous URI.
 # @category Source / Input
+# @flag extra
```
Comment (smimram) [inline libs/playlist.liq]: This one looks pretty important to me.
```suggestion
```

File: libs/video.liq [hunk 3]
```diff
@@ -18,6 +18,7 @@ end
 
 # Add a static image on the first video channel.
 # @category Source / Video Processing
+# @flag extra
```
Comment (smimram) [inline libs/video.liq]: I expect that "many people" will use this in order to add logos.
```suggestion
```

File: libs/hls.liq (hunk 1 above)
Comment (toots) [reply libs/hls.liq]:
```suggestion
# @flag experimental
```

**Pull request merged**